### DISCUSSION TITLE
Using `useSortedClasses` with Svelte

### DISCUSSION THREAD

**Discussion by beeb** [biomejs/biome#3559] [Q&A] [answered]
Has anyone managed to have the `useSortedClasses` lint work with svelte files? I have plenty of tailwind classes in random order but the lint never fires.

Config:

```json
{
	"$schema": "https://biomejs.dev/schemas/1.8.3/schema.json",
	"organizeImports": {
		"enabled": true
	},
	"formatter": {
		"enabled": true,
		"lineWidth": 120
	},
	"javascript": {
		"formatter": {
			"quoteStyle": "single",
			"semicolons": "asNeeded"
		}
	},
	"linter": {
		"enabled": true,
		"rules": {
			"nursery": {
				"useSortedClasses": "error"
			}
		}
	},
	"overrides": [
		{
			"include": ["*.svelte"],
			"linter": {
				"rules": {
					"style": {
						"useConst": "off"
					}
				}
			}
		}
	]
}
```

**Comment by ematipico** [comment, marked as answer]
I believe you missed the documentation that explains the **partial** support of svelte files: https://biomejs.dev/internals/language-support/#html-super-languages-support

**Reply by beeb** [reply]
Got it, so it only lints the part where there are no HTML classes, and doesn't lint `.html` files either. So it's mostly a lint for React then?

**Reply by notramo** [reply]
It's currently only for frameworks which use JSX e.g. Solid, Qwik, and React. However, Svelte adds custom HTML syntax, so it's not supported yet.
